### PR TITLE
Added few libraries

### PR DESCRIPTION
**Pull request by Ruturaj2728**
# 📝 Why & how

This PR adds the following libraries to the directory:

https://www.npmjs.com/package/@mindinventory/react-native-tab-bar-interaction
https://www.npmjs.com/package/@mindinventory/rn-top-navbar
https://www.npmjs.com/package/@mindinventory/react-native-card-animation
https://www.npmjs.com/package/@mindinventory/react-native-stagger-view
https://www.npmjs.com/package/@mindinventory/react-native-boilerplate
https://www.npmjs.com/package/@mindinventory/react-native-speed-view

# ✅ Checklist

- [ ] Added library to **`react-native-libraries.json`**
- [ ] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

**Review by Simek — APPROVED** [review]
Thank you for adding several new entries, LGTM! 👍 

Please resolve the conflict and this should be ready to ship.

**Comment by Ruturaj2728** [comment]
@Simek , conflicts resolved.

**Comment by Simek** [comment]
@Ruturaj2728 Thank you, but it looks like merge commit have removed library added in previous PR:
* https://github.com/Mindinventory/awesome-clock

Can you correct that? It should be placed above new entries, since we use this order for "Recently added" sort option.

File: react-native-libraries.json [hunk 1]
```diff
@@ -8716,6 +8716,70 @@
       "https://user-images.githubusercontent.com/87525902/181426355-eb55b0e1-dc2e-484f-8675-f699e61bea80.gif"
     ],
     "ios": true,
+    "android": true,
```
Comment (Simek) [inline react-native-libraries.json:8719]:
```suggestion
    "android": true
```

Fix syntax error.

**Pull request merged**